bay/images/clamav/readiness.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/sh
+
+set -e
+
+if freshclam | grep -q 'bytecode.cvd database is up to date'; then
+  echo 'freshclam running successfully'
+  if clamdscan eicar.com | grep -q 'Infected files: 1'; then
```

Comment:
Is this actually testing that the service is up and accepting connections though?